hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -21,8 +21,9 @@ public record ContractsConfig(
         @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerTransaction,
         @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerSec,
         @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerSecBackend,
-        @ConfigProperty(defaultValue = "500000000") @NetworkProperty long maxOpsDuration,
-        @ConfigProperty(defaultValue = "1") @NetworkProperty int opsDurationThrottleBurstSeconds,
+        @ConfigProperty(defaultValue = "500000000") @NetworkProperty long opsDurationThrottleCapacity,
+        @ConfigProperty(defaultValue = "500000000") @NetworkProperty long opsDurationThrottleUnitsFreedPerSecond,
+        @ConfigProperty(defaultValue = "1") @NetworkProperty long opsDurationThrottleUnitsRequiredToExecute,
```

Comment:
we might consider not using a configuration parameter for this.  I can't think of a reason why we would want to update this at runtime?  It will add to the clutter of configs we need to set when we eventually make this feature available.